frappe/commands/utils.py

```diff
@@ -782,12 +782,11 @@ def run_tests(
 			click.secho(f"bench --site {site} set-config allow_tests true", fg="green")
 			return
 
-		frappe.init(site=site)
-
 		frappe.flags.skip_before_tests = skip_before_tests
 		frappe.flags.skip_test_records = skip_test_records
```

Comment:
@vorasmit You're right. This makes it so that the `.flags` is still uninitialized.

These flags have no inherent natural semantic dependencies on an initialized site. Let me think how to fix this properly.